docs/developer/app-store/apps/product-feed.mdx

```diff
@@ -0,0 +1,139 @@
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.tsx";
+
+# Product feed
+
+## Metadata
+
+<AppMetadata
+  minSaleorVersion="3.10"
+  roadmapUrl="https://github.com/saleor/apps/labels/App%3A%20Product%20Feed"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/products-feed"
+/>
+
+## Introduction
+
+The app generates XML files in Google Merchant feed format.
+
+## Features
+
+- Generate feed in the [Google Merchant format](https://support.google.com/merchants/answer/7439882)
+- Map your catalog structure to Google Merchant format
+- Upload files to the AWS S3 bucket
+- Separate feed for each of the channels
+
+## Assumptions & limitations
+
+- Each item in the feed is equivalent to a single Product Variant in Saleor
+- For catalogs bigger than 200 product variants an AWS S3 bucket is recommended
```

Comment:
recommended? I think it will just not work

Reply:
Update: after the infra update, the app will handle much more - tested successfully on 5k products, without the s3. Dont have a data for bigger collections.

- Would consider adding a hard limit 200 items when 3s is not configured to the app to minimize traffic costs on lambdas.
- Would consider adding limit ~50k products to the app (would require a test) to manage the expectations to the app performance and capabilities. Current hosting setup has an upper limit of compute time.